api/src/services/__tests__/competenciesService.ts

```diff
@@ -0,0 +1,19 @@
+import { listCompetencies } from '../competenciesService';
+import { mockQuery } from '../mockDb';
+
+describe('listCompetencies', () => {
+  it('should load a page that lists the competencies in the database with their titles and descriptions.', async () => {
+    mockQuery(
+      'select `id`, `label`, `description` from `competencies` limit ? offset ?',
+      [10, 10],
+      [{ id: 1, label: 'Code', description: 'To code or not to code...' }]
+    );
+    expect(await listCompetencies(10, 10)).toEqual([
+      {
+        id: 1,
+        label: 'Code',
+        description: 'To code or not to code...',
```

Comment:
Would it be valuable to test a real-world example of a comptency here instead of "Code"?

Reply:
We shall change it to general statements like: 
label: "Competency name goes here"
description: "Competency description goes here"

Reply:
I'm looking at other files and I think the convention set is to have the values without the "goes here" text.

Reply:
Why not follow the pattern from other files? https://github.com/OpenTree-Education/rhizone-lms/blob/main/api/src/services/__tests__/reflectionsService.ts#L38

Reply:
Yes we now follow the pattern

Reply:
Yes we now follow the pattern

Reply:
Yes we now follow the pattern

Reply:
Yes we now follow the pattern